tdrs-frontend/nginx/README.md

```diff
@@ -0,0 +1,88 @@
+# Nginx Documentation
+
+## 1. Installation
+
+The Nginx uses a master process and mutiple workers to efficiently manage requests. The master process reads and evaluates config file and manages the workers. 
+
+In TDRS, all config and associated files with it are placed in ```/tdrs-frontend/nginx``` folder. There are various differences between the local and deployed versions. Due to the fact that cloud.gov uses buildpacks and local version uses containerized docker instance, there are two separate directories for local and deployed versions.
+
+## 2. Architecture
+
+In general for TDRS, the *Nginx* handles incoming HTTP requests to both frontend and backend. The frontend requests are handled directly while the backend requests are forwarded to the backend server.
+
+![Cloud.gov Architecture](./src/arch1.jpg)
```

Comment:
We might be able to keep a single diagram and just update this one:

https://github.com/raft-tech/TANF-app/blob/develop/docs/Security-Compliance/boundary-diagram.md